src/Presentation/DonationConfirmationPageSelector.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Presentation;
+
+/**
+ * Selects a random confirmation page from given options
+ *
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class DonationConfirmationPageSelector {
+
+	private $config;
+
+	public function __construct( array $config ) {
+		$this->config = $config;
+	}
+
+	public function selectPage(): string {
+		return empty( $this->config ) ? '' : $this->config[$this->getRandomIndex()];
```

Comment:
The configuration will be extended by several values anyway. How about defining the default confirmation page in the config and returning that? We would also get rid of the hard coded one in DonationConfirmation.twig:4